charts/sonarqube-dce/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ All changes to this chart will be documented in this file.
 * Fix the postgresql chart's repository link
 * Calibrate probes' initialDelaySeconds
 * Define HorizontalPodAutoscaler for the app deployment
+* Add ApplicationNodes.affinity, ApplicationNodes.tolerations, searchNodes.affinity, searchNode.tolerations, ApplicationNodes.nodeSelector, searchNodes.nodeSelector
```

Comment:
Can we reduce it to "Define affinity, tolerations, and nodeSelector parameters for App and Search nodes"